.buildkite/scripts/run_models.sh

```diff
@@ -16,12 +16,21 @@ db=$1
 echo `pwd`
 cd integration_tests
 dbt deps
+if [ "$db" = "databricks-sql" ]; then
+dbt seed --vars '{salesforce_marketing_cloud_schema: sfmc_sqlw_tests}' --target "$db" --full-refresh
+dbt run --vars '{salesforce_marketing_cloud_schema: sfmc_sqlw_tests}' --target "$db" --full-refresh
+dbt test --vars '{salesforce_marketing_cloud_schema: sfmc_sqlw_tests}' --target "$db"
+dbt run --vars '{salesforce_marketing_cloud_schema: sfmc_sqlw_tests}' --target "$db"
+dbt test --vars '{salesforce_marketing_cloud_schema: sfmc_sqlw_tests}' --target "$db"
+dbt run --vars '{salesforce_marketing_cloud_schema: sfmc_sqlw_tests, salesforce_marketing_cloud__link_enabled: false, salesforce_marketing_cloud__list_enabled: false}' --full-refresh --target "$db"
```

Comment:
Should there be a dbt run (incremental) after the full-refresh, similar to `fivetran_log`? ([link](https://github.com/fivetran/dbt_fivetran_log/blob/main/.buildkite/scripts/run_models.sh#L49))

Reply:
Yeah including an incremental run after a full refresh would be best here. Added to both sql warehouse and normal sections of the run_models.sh